.github/workflows/test_workflows_synchronization.yml

```diff
@@ -0,0 +1,16 @@
+name: Test Workflows Synchronization
+
+on:
+  pull_request:
+    paths:
+    - '.github/workflows/**'
+
+jobs:
+  test-doc:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Tests
+      run: |
+        if [[ "$(diff .github/workflows/test_suite_linux.yml .github/workflows/test_suite_linux_develop.yml | grep -c -e \"^<\" -e \"^>\")" -ne "13" ]]; then echo Linux Workflows not synchronized; exit -1; fi
```

Comment:
Checking the number of diff lines is quite simple, but should be sufficient to detect when one workflow is updated and not the other.

Reply:
Not sure if it is better to create a dedicated workflow or merge it with the 'source' one?